validated/views.py

```diff
@@ -400,6 +400,8 @@ class DataReport(FormView):
     success_url = "/validated/data_report/"
 
     def post(self, request, *args, **kwargs):
+        from .dash_apps.finished_apps import data_report
```

Comment:
I'm not entirely sure, but this is a `post` request, so it not executed just when requesting the page, but when selecting something in the existing form (the station, variable, etc.) and asking for the report. As such, when you just load `/validated/data_report/` for the first time, the `DataReportDash` has not been created.

Maybe you need to create your own `get` method for this `DataReport` form view in where to have that import statement?